examples/covariance/plot_robust_vs_empirical_covariance.py

```diff
@@ -40,16 +40,8 @@
   so one can trust our implementation by comparing to this case.
 
 
-References
-----------
-.. [1] P. J. Rousseeuw. Least median of squares regression. Journal of American
-    Statistical Ass., 79:871, 1984.
-.. [2] Johanna Hardin, David M Rocke. The distribution of robust distances.
```

Comment:
Oh. I had expected that these would be present in the class docs. Apparently not.

Reply:
@VirgileFritsch, it's unclear to the reader what the relevance of these references in `plot_robust_vs_empirical_covariance.py` is, since they are not cited anywhere. Are these references here, rather than in `MinCovDet`, because they are the inspiration for the example?